src/runtime/BigInt.cpp

```diff
@@ -31,7 +31,9 @@ BigIntData::~BigIntData()
 
 BigIntData::BigIntData(BigIntData&& src)
 {
+    bf_init(src.m_data.ctx, &m_data);
     bf_move(&m_data, &src.m_data);
+    bf_init(m_data.ctx, &src.m_data);
```

Comment:
I'm not sure why the first `bf_init` should be invoked before `bf_move`.
Does `m_data` needs to be initialized before move operation?

Reply:
bf_move function is not `move` contents of bf_t...
bf_move function just copy bf_t's content so we have to reset src content